### PR TITLE
fix: all windows l10n

### LINKED ISSUE #3: Accounts Window Localization
Currently, localization works only in the Talk app, but doesn't work on the Accounts window.

- [x] Detect user OS locale before login
- [ ] ? Reuse localization from Nextcloud login web page ?
- [x] Add l10n for the Accounts window
- [ ] Add l10n to Login WebView (in headers)

### LINKED ISSUE #363: The language is in English instead of German
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are interested into the same feature.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

---

If I start NC Talk Desktop it is in English but, I want to use it in german.

### PR DESCRIPTION
**Pull request by ShGKme**
### ☑️ Resolves

* Fix #3 
* Fix #363

### 🖼️ Screenshots

image

P.S. Translated by GitHub Copilot

### 🚧 Tasks

- [x] Add `app:getSystemL10n` to get the preferred language and system locale
- [x] Add universal `setupWebPage` util to be used to set up every window, including l10n registration
- [x] Use system preferred language on Accounts window
- [x] Add missing t-strings

### CODE REVIEW

**Review by nickvergessen — APPROVED** [review]

**Comment by ShGKme** [comment]
Also added language preferences on login web-view

image

File: src/authentication/renderer/AuthenticationApp.vue [hunk 1]
```diff
@@ -152,22 +152,28 @@ export default {
...
+					foundVersion: capabilitiesResponse.version.string,
+				}))
 			}
 			if (parseInt(talkCapabilities.version.split('.')[0]) < MIN_REQUIRED_TALK_VERSION) {
 				// TODO: use semver package and check not only major version?
-				return this.setError(`Nextcloud Talk ${MIN_REQUIRED_TALK_VERSION} or higher is required but ${talkCapabilities.version} is installed`)
+				return this.setError(t('talk_desktop', `Nextcloud Talk {minRequiredVersion} ${MIN_REQUIRED_TALK_VERSION} or higher is required but {foundVersion} ${talkCapabilities.version} is installed`, {
+					minRequiredVersion: MIN_REQUIRED_TALK_VERSION,
+					foundVersion: talkCapabilities.version,
+				}))
```
[the remark this answers is not in the document]
Comment (ShGKme) [reply src/authentication/renderer/AuthenticationApp.vue:176]: The error above is about the Nextcloud server and this one is about Nextcloud Talk.

But we can replace `Nextcloud [Talk]` with translation parameter `{ComponentName}` to decrease the amount of translation keys.

[the remark this answers is not in the document]
Comment (ShGKme) [reply src/authentication/renderer/AuthenticationApp.vue:176]: Oh, I see 🙈

[the remark this answers is not in the document]
Comment (ShGKme) [reply src/authentication/renderer/AuthenticationApp.vue:176]: There should be no template string

[the remark this answers is not in the document]
Comment (ShGKme) [reply src/authentication/renderer/AuthenticationApp.vue:176]: Nice catch, thanks!

**Review by DorraJaouad — APPROVED** [review]

**Pull request merged**